README.md

```diff
@@ -284,6 +284,8 @@ curl -N 127.0.0.1:8080/generate_stream \
     -H 'Content-Type: application/json'
 ```
 
+Currently the model cannot support the multi-cards inference.
```

Comment:
Currently Llava-v1.16 inference on multi-cards is not supported.

Reply:
Done